sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveDataFrameAnalyticsSuite.scala

```diff
@@ -74,4 +78,10 @@ class HiveDataFrameAnalyticsSuite extends QueryTest with TestHiveSingleton with
       sql("select a, b, sum(b) from mytable group by a, b with cube").collect()
     )
   }
+
+  // There was a bug in DataFrameFrameReader.table and it has problem for table with schema name,
+  // Before fix, it throw Exceptionorg.apache.spark.sql.catalyst.analysis.NoSuchTableException
+  test("table name with schema") {
+    hiveContext.read.table("usrdb.test")
```

Comment:
Sorry, I think I was unclear.  I don't think we should put this in `HiveDataFrameAnalyticsSuite` since it has nothing to do with analytics.   I was just suggesting to use this as a model.  Lets just make a separate generic `HiveDataFrameSuite` since this is pretty core functionality.

While we are at it.  I'm also not a huge fan of using beforeAll and afterAll for test setup for a single test since it means the state is spread out across the file (sorry for the bad example).  I would just do it all in the test block.

For the description in the comments, its customary to link to the JIRA.